project/src/main/webapp/document.jsp

```diff
@@ -233,6 +234,18 @@
         return false;
       }
 
+      function deleteDoc() {
+        var hash = "<%= (String)request.getAttribute("documentHash") %>"
+        var xhttp = new XMLHttpRequest();
+        xhttp.open("GET", "/DeleteDoc?documentHash=" + hash, true);
+        xhttp.onreadystatechange = function() {
```

Comment:
not sure if we need the console to log deleted once done? theoretically the frontend would reflect it

Reply:
testing reasons